backend/consensus/base.py

```diff
@@ -258,7 +279,7 @@ async def exec_transaction(
 
             if (
                 len([vote for vote in votes.values() if vote == Vote.AGREE.value])
-                >= num_validators // 2
+                >= (num_validators + 1) // 2
```

Comment:
@cristiam86 @AgustinRamiroDiaz Please, look carefully at this change. It ceils instead of flooring to have "majority" as per spec. Also, I am not sure about how to handle execution errors correctly. right now leader on error (i.e. ill formed contract) says that it disagrees, and without this change ill-formed contract will be deployed....

Reply:
Regarding the majority, why did the previous comparison don't work? 
2 >= 2.5 (no majority reached)
3 >= 2.5 (majority reached)

Reply:
Regarding the execution errors, the leader can't disagree. They all should agree in the execution error that happened.
The contract being deployed seems like a previous bug

Reply:
I think that something that it's not as per the spec now is that the leader is not running as a validator later, and just voting "AGREE". 

I've created https://github.com/yeagerai/genlayer-simulator/issues/620 to track this